### PR TITLE
Progress: misc launch-blocking fixes

### PR DESCRIPTION
**Pull request by uxengine**
Misc fixes:
- don't restore document position at 100%
- display max percentage in teasers if it's 100%
- enable progress circle on article teasers also on Bookmarks page
- add more user guidance and visibility to back-to-top button
- load mediaProgress independent of document (so that audio progress refreshes cross-device on close and re-open)

Depends on https://github.com/orbiting/styleguide/pull/219

### CODE REVIEW

File: lib/translations.json [hunk 1]
```diff
@@ -3048,7 +3048,23 @@
...
+      "key": "article/progress/time/today",
+      "value": "von heute"
+    },
+    {
+      "key": "article/progress/time/yesterday",
+      "value": "von gestern"
+    },
+    {
+      "key": "article/progress/time/other",
+      "value": "vom {date}"
```
Comment (patrickvenetz) [inline lib/translations.json]:
```suggestion
      "value": "am {date}"
```

Schweizer nutzen "*am Tag.Monat.Jahr"

Comment (uxengine) [reply lib/translations.json]: Merci vielmal. Isch jetz helvetisiert, au de Tescht.

File: components/Article/Progress/index.js [hunk 2]
```diff
@@ -333,8 +321,19 @@ class Progress extends Component {
     }, 5000, { 'trailing': false })
 
     this.getMediaProgress = (mediaId) => {
-      const { mediaProgress } = this.state
-      return mediaProgress ? mediaProgress[mediaId] : undefined
+      return new Promise((resolve, reject) => {
```
Comment (tpreusse) [inline components/Article/Progress/index.js]: directly return `this.props.client.query().then()`, `this.props.client.query` already returns a promise, and `then` on a promise also always returns another promise.

No `catch` here, it's fine to swallow the error in the audio / video player but IMO no need to already swallow here. Also in the current implementation if `data.mediaProgress || data.mediaProgress === null` does not pass the promise is never resolved or rejected.

Comment (tpreusse) [reply components/Article/Progress/index.js]: API docs on `errors` is not super clear, at least I didn't find it. Therefore just ignore, I guess. In theory the promise should also be rejected.

**Pull request merged**